NSS-Day-Cohort-42/rare-media-moguls
# User Nav completed

#### Changes Made
1. Modified file `Nav.js` to include dropdown list with all required link functionality.
1. Modified `ApplicationViews.js` to allow Nav to be sticky at the top of all views

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-user-nav
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> When user is logged in, the Nav should be present at the top of the screen. 
> When user clicks the square icon in the top right corner, additional menu items should appear in a dropdown. 
> Links are not live at this time, but should be routed as expected per wireframe.
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
